lib/zebra.h

```diff
@@ -512,6 +512,26 @@ typedef uint32_t route_tag_t;
 #define ROUTE_TAG_MAX UINT32_MAX
 #define ROUTE_TAG_PRI PRIu32
 
+static inline const char *iana_afi2str(iana_afi_t afi)
```

Comment:
how does this function materially differ from aft2str, bgp_afi and iana_afi should be the same....

Reply:
perhaps have the new function map to the internal AFI value and call afi2str?